src/test/java/guitests/PersonEditGuiTest.java

```diff
@@ -52,21 +51,20 @@ public void editPerson_usingAccelerator() {
         assertEquals(alicePersonCard, newAlice);
         assertTrue(personListPanel.isSelected(newAlice));
 
-
         //Confirm right values are displayed after grace period is over
         sleepForGracePeriod();
         assertEquals(alicePersonCard, newAlice);
 
+        //Confirm cancel operation does not cancel the edit after the grace period.
+        personListPanel.use_PERSON_CHANGE_CANCEL_ACCELERATOR();
```

Comment:
This is not completely correct. The cancel is inserted between two asserts with two different purposes. By right, you should do both asserts first, cancel, and repeat both asserts again.